test_scripts/API/VehicleData/SubscribeVehicleData/003_RPC_parameter_already_subscribed_Result_IGNORED_flow.lua

```diff
@@ -71,7 +71,7 @@ local vehicleDataResults2 = {
     resultCode = "DATA_ALREADY_SUBSCRIBED"
   },
   electronicParkBrakeStatus = {
-    dataType = "VEHICLEDATA_TIREPRESSURE", 
+    dataType = "VEHICLEDATA_ELECTRONICPARKBRAKESTATUS", 
```

Comment:
@YarikMamykin This fix is not related to issue, though lets not remove it

Reply:
@dboltovskyi , fixed in 9bb69122b2f3b293411f545c171aa276625320f3